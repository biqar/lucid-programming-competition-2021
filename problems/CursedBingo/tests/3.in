8
243
1235
4002
42
920001
31239
123319
1238122