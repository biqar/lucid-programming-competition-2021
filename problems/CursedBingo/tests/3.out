58806
1523990
16012002
1722
846400920000
975843882
15207452442
1532944848762